egs/aspire/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -52,21 +45,34 @@ if [ $stage -le 1 ]; then
     else
       num_reps=1
     fi
-
-    python steps/data/reverberate_data_dir.py \
-      --prefix "rev" \
-      --rir-set-parameters "1.0, SIM_RIR/simulated_rir_mediumroom/rir_list" \
-      --noise-set-parameters musan/noise/free-sound/noise_list \
-      --foreground-snrs $foreground_snrs \
-      --background-snrs $background_snrs \
-      --speech-rvb-probability 1 \
-      --pointsource-noise-addition-probability 1 \
-      --isotropic-noise-addition-probability 1 \
-      --rir-smoothing-weight 0 \
-      --num-replications $num_reps \
-      --max-noises-per-minute 1 \
-      --random-seed 1 \
-      data/${data_dir} data/${data_dir}_rvb
+    if [ "$base_rirs" == "simulated" ]; then
+      python steps/data/reverberate_data_dir.py \
+        --prefix "rev" \
+        --rir-set-parameters "0.5, RIRS_NOISES/simulated_rirs/smallroom/rir_list" \
+        --rir-set-parameters "0.5, RIRS_NOISES/simulated_rirs/mediumroom/rir_list" \
+        --noise-set-parameters RIRS_NOISES/pointsource_noises/noise_list \
+        --foreground-snrs $foreground_snrs \
+        --background-snrs $background_snrs \
+        --speech-rvb-probability 1 \
+        --pointsource-noise-addition-probability 1 \
+        --isotropic-noise-addition-probability 1 \
+        --rir-smoothing-weight 0 \
+        --num-replications $num_reps \
+        --max-noises-per-minute 1 \
+        --random-seed 1 \
+        data/${data_dir} data/${data_dir}_rvb
+    else
+      reverb_data_dirs=
```

Comment:
Once again, it is not necessary to keep the old scripts. Just provide the appropriate options for using the new scripts to generate the old results (at least approximately). I think nothing changed other than the fact that you are using new rir and noise sets.